component/Map.js

```
@@ -1,59 +1,497 @@
 import tw from "tailwind-styled-components/dist/tailwind";
 import mapboxgl from "mapbox-gl";
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
+import SimpleMap from "./SimpleMap";
 
 const Map = ({ pick, drop }) => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)